packages/tests/tests/multiaddr.node.spec.ts

```diff
@@ -1,6 +1,5 @@
 import type { PeerId } from "@libp2p/interface";
 import type { PeerInfo } from "@libp2p/interface";
-import { CustomEvent } from "@libp2p/interface";
```

Comment:
is this change due to `nodejs@20`/browser having `CustomEvent` available?

Reply:
Indeed. It's now included part of the engine